tdrs-backend/tdpservice/parsers/test/conftest.py

```diff
@@ -0,0 +1,557 @@
+"""Package level fixtures."""
+import pytest
+from tdpservice.data_files.models import DataFile
+from tdpservice.parsers.test.factories import DataFileSummaryFactory, ParsingFileFactory
+from tdpservice.parsers import util
+
+@pytest.fixture
+def small_correct_file(stt_user, stt):
```

Comment:
this is not a change request to your PR, but maybe a discussion point: I think we should start using the util function directly in our tests instead of creating fixture. It will be more readable.